### PR TITLE
sustainability

### PR DESCRIPTION
**Pull request by yelghali**
# READ CAREFULLY Thanks for contributing !! :)

## Branch staging

Do a merge on the **Staging branch**, NOT master. Github Actions does not allow to share secrets so during a PR, I can only do a build validation. I need an in-between branch to be able to deploy on a staging environment (https://stoakscheckliststaging.z16.web.core.windows.net/)

## Translation

If you added a translation, please ensure that you didn't forget anything:

- copy the folder **/data/en** and translate all information
- in the localized files, modify the URL to target your langage (i.e: docs.microsoft.com/**YOURLANG**/link)
- copy the file **src/views/en.html** and translate it
- ensure that a flag is existing for your language (**/src/img/flags**)
- add a link for your lang in the header (**src/view/base/header.pug**)
- add your name to contributors

### CODE REVIEW

File: data/en/items/container.json [hunk 1]
```diff
@@ -54,6 +55,23 @@
...
+      {
+        "title": "Caching for ACR",
+        "url": "https://learn.microsoft.com/en-us/azure/container-registry/tutorial-registry-cache"
+      }
+    ],
+    "tags": [
+      "all",
+      "sustainability",
+      "performance",
+      "security"
```
Comment (lgmorand) [inline data/en/items/container.json:73]: security ? :)

Comment (yelghali) [reply data/en/items/container.json]: typo ;) fixed

**Comment by yelghali** [comment]
@HoussemDellai  @lgmorand  @mosabami for review pls

**Comment by lgmorand** [comment]
@yelghali  t'as des conflicts (de canard)

**Comment by lgmorand** [comment]
@yelghali faut que tu la corriges rapidement ou alors je vais devoir la clore, ca me bloque pour gérer les branches et migrer le repo :)

**Comment by lgmorand** [comment]
@yelghali celle là on la ferme, on est d'accord ?

**Comment by lgmorand** [comment]
pas de réponse @yelghali , moi je ferme :)

**Pull request closed without merging**